mmv1/third_party/terraform/website/docs/d/oracle_database_db_nodes.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Oracle Database"
+description: |-
+  List all database nodes of a Cloud VmCluster.
+---
+
+# google_oracle_database_db_nodes
+
+List all DbNodes of a Cloud VmCluster.
```

Comment:
Please add a link to the cloud.google.com REST API documentation for this endpoint.

Reply:
This is read only resource, added API link. Thank you for the suggestions.